stdlib/pagerduty/pagerduty_test.flux

```diff
@@ -4,24 +4,24 @@ import "array"
 import "pagerduty"
 import "testing"
 
-input = array.from(
+input = () => array.from(
```

Comment:
why the change to making it a function? Does it have any effect on the result?

Reply:
Forgot to respond to this. No, I thought it did so I changed it and it didn't have an effect. Then I forgot to put it back. This works nicely anyway though.